### PR TITLE
client: set fiat auth user for delete pipelines request 

### PR DESCRIPTION
**Pull request by its-aqeel**
### Summary of Changes

all request going via wrapper `request` but delete function deleting stale pipelines directly and was missing header for fiat user

### PR Checklist

Make sure the following checklist items are done before merging this PR.

- [x] Update `CHANGELOG.md` in the `## Unreleased` section ([instructions here])
- [x] Make sure tests are passing

[instructions here]: https://keepachangelog.com/en/1.0.0/#how

### CODE REVIEW

**Review by jasonmcintosh — APPROVED** [review]

File: client.go [hunk 1]
```diff
@@ -280,6 +280,10 @@ func (c *Client) Delete(url, traceparent string) error {
 	if err != nil {
 		return err
 	}
+	if c.FiatUser != "" {
+	   request.Header.Set(SpinFiatUserHeader, c.FiatUser)
+	   request.Header.Set(SpinFiatAccountHeader, c.FiatUser)
```
Comment (jasonmcintosh) [inline client.go:285]: I've temporarily approved but TECHNICALLY account header isn't the right spot for the fiat user.  SHouldn't set that JUST the user header.

Comment (jasonmcintosh) [reply client.go:285]: ALSO be kinda nice if a test on this...

Comment (its-aqeel) [reply client.go:285]: tested in private environment and it's working fine

**Comment by jasonmcintosh** [comment]
Closing in favor of another PR which adds this.

**Pull request closed without merging**